rxandroidble/src/main/java/com/polidea/rxandroidble/internal/connection/MtuWatcher.java

```diff
@@ -0,0 +1,45 @@
+package com.polidea.rxandroidble.internal.connection;
+
+
+import java.util.concurrent.atomic.AtomicInteger;
+import javax.inject.Inject;
+import javax.inject.Named;
+import rx.Completable;
+import rx.CompletableSubscriber;
+import rx.Subscription;
+import rx.functions.Action1;
+
+@ConnectionScope
+class MtuWatcher extends Completable implements MtuProvider {
```

Comment:
MtuWatcher or just MtuCache?

Reply:
Currently it is watching the Mtu and providing the current value, name may be changed

Reply:
ok